### DISCUSSION TITLE
`[notice]` attribute

### DISCUSSION THREAD

**Discussion by ghost** [vlang/v#14629] [Ideas]
It would be nice to have an attribute that does not make it `[deprecated]` but gives a warning.
For example, `[$if android: notice: 'must not be used on Android, use xxxxxx() instead']`.

#13497

**Comment by dumblob** [comment]
Can't you just write the same thing using the compile-time computation abilities? Why would *V* need to introduce yet another attribute? Why to break the "one way to do a thing" mantra?

I mean something like `$if android { assert 'must not be used on Android, use xxxxxx() instead' == '' }`?

**Comment by dumblob** [comment]
> Can't you just write the same thing using the compile-time computation abilities? Why would _V_ need to introduce yet another attribute? Why to break the "there is only one way to do a thing" mantra?
> 
> I mean something like `$if android { assert 'must not be used on Android, use xxxxxx() instead' == '' }`?
